Components/badge.css

```diff
@@ -0,0 +1,55 @@
+/* BADGE COMPONENT */
+
+/* Badge on Avatar */
+
+.badge-on-avatar {
+  position: relative;
+  width: max-content;
+}
+
+.badge-on-icon {
+  position: relative;
+  width: max-content;
+  padding: 1rem;
+}
+
+.badge-on-icon i {
+  font-size: 1.5rem;
+}
+
+.badge {
+  font-family: inherit;
```

Comment:
move this to global declarations / common / resets

Reply:
I took this line out. Just like in the button component, font-family is already inherited so no need for this. thanks for the review